www/js/lib/uiUtil.js

```diff
@@ -406,9 +406,15 @@ define(rqDef, function(settingsStore) {
      * Removes the animation effect between various sections
      */
     function removeAnimationClasses() {
-        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
-        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
-        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
+        let configuration = document.getElementById('configuration');
```

Comment:
@dheerajdlalwani There is still one more `let` statement here.

Reply:
Fixed it.